image-builder.spec

```diff
@@ -62,6 +62,14 @@ install -m 0644 -vp internal/db/migrations/*          %{buildroot}%{_datadir}/im
 %if %{with tests} || 0%{?rhel}
 install -m 0755 -vd					%{buildroot}%{_libexecdir}/tests/image-builder
 install -m 0755 -vp test/cases/*                        %{buildroot}%{_libexecdir}/tests/image-builder/
+
+install -m 0755 -vd                                             %{buildroot}%{_datadir}/tests/image-builder/edge
+install -m 0644 -vp test/data/edge/*                            %{buildroot}%{_datadir}/tests/image-builder/edge/
+
+install -m 0755 -vd                                             %{buildroot}%{_datadir}/tests/image-builder/keyring
+install -m 0644 -vp test/data/keyring/id_rsa.pub                %{buildroot}%{_datadir}/tests/image-builder/keyring/
+install -m 0600 -vp test/data/keyring/id_rsa                    %{buildroot}%{_datadir}/tests/image-builder/keyring/
```

Comment:
Isn't this redundant? 

Aren't commands on line 66-67 going to install the keyring as well?

Reply:
line 66-67 is to copy data files under test/edge, and this two lines is to copy key files under test/keyring.
I mean, I can put these files together but I'd like to follow the same dir path in osbuild-composer.

Reply:
And why not copy `test/data/*`  under `%{buildroot}%{_datadir}/tests/image-builder/data/` instead ?

Reply:
Sure, I will update my code if you suggest so. I just followed the dir hierarchy of osbuild-composer project.

Reply:
One more question, I will copy test/data/* under %{buildroot}%{_datadir}/tests/image-builder/data/, but need to chmod rsa private key to 600.

Currently I am doing this:
install -m 0600 -vp test/data/keyring/id_rsa                    %{buildroot}%{_datadir}/tests/image-builder/keyring/

And if I change to this, how to chmod id_rsa file?
install -m 0755 -vd                                             %{buildroot}%{_datadir}/tests/image-builder/data
install -m 0644 -vp test/data/*                                 %{buildroot}%{_datadir}/tests/image-builder/data/

Reply:
Argh. It's too much to bother. Leave it like it is.